trestle/core/commands/merge.py

```diff
@@ -14,119 +14,131 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Trestle Merge Command."""
-
+import argparse
+import logging
+import os
 from pathlib import Path
-from typing import Set, Type
 
 from ilcli import Command  # type: ignore
 
+import trestle.core.err as err
 from trestle.core import const, utils
-from trestle.core.base_model import OscalBaseModel
-from trestle.utils import fs
+from trestle.core.models.actions import CreatePathAction, RemovePathAction, WriteFileAction
+from trestle.core.models.elements import Element, ElementPath
+from trestle.core.models.file_content_type import FileContentType
+from trestle.core.models.plans import Plan
+from trestle.utils import fs, load_distributed
 from trestle.utils import log
 
+logger = logging.getLogger(__name__)
+
 
 class MergeCmd(Command):
     """Merge subcomponents on a trestle model."""
 
     name = 'merge'
 
     def _init_arguments(self) -> None:
-        self.add_argument('-e', '--elements', help='Comma-separated list of paths of properties to be merged.')
         self.add_argument(
-            '-l',
-            '--list-available-elements',
-            action='store_true',
-            help='Comma-separated list of paths of properties that can be merged.'
+            f'-{const.ARG_ELEMENT_SHORT}',
+            f'--{const.ARG_ELEMENT}',
+            help=f'{const.ARG_DESC_ELEMENT}(s) to be merged. The last element is merged into the second last element.',
+            required=True
         )
 
-    def _run(self, args) -> None:
+    def _run(self, args: argparse.Namespace) -> int:
         """Merge elements into the parent oscal model."""
         log.set_log_level_from_args(args)
-        if args.list_available_elements:
-            self._list_available_elements()
-
-    def _list_available_elements(self) -> None:
-        """List element paths that can be merged from the current context."""
-        current_model, current_alias = fs.get_contextual_model_type(Path.cwd())
-
-        current_filename = f'{current_alias}.json'
-
-        self._list_options_for_merge(Path.cwd(), current_alias, current_model, current_filename)
-
-    def _list_options_for_merge(
-        self,
-        cwd: Path,
-        current_alias: str,
-        current_model: Type[OscalBaseModel],
-        current_filename: str,
-        initial_path: Path = None,
-        visited_elements: Set[str] = None
-    ):
-        """List paths that can be used in the -e option for the merge operation."""
-        if initial_path is None:
-            initial_path = cwd
-        if visited_elements is None:
-            visited_elements = set()
-
-        path_sep = '.' if current_alias else ''
-
-        # List options for merge
-        if not utils.is_collection_field_type(current_model):
```

Comment:
Do we think this is useful? Should it be added to a backlog?

Reply:
Hmm... it could be useful. However, I did not investigate. May be we can keep the branch alive and bring back in future release?